packages/loader/driver-utils/src/blobCacheStorageService.ts

```diff
@@ -17,12 +17,8 @@ export class BlobCacheStorageService extends DocumentStorageServiceProxy {
         super(internalStorageService);
     }
 
-    public async read(id: string): Promise<string> {
+    public async readBlob(id: string): Promise<ArrayBufferLike> {
         const blob = this.blobs.get(id);
```

Comment:
another example blob is string

Reply:
Have you had a chance to trace where blobs are coming from and who creates them?
My quick search reviles that all callers (directly or indirectly) do  buffer.toString("base64"), which is a waste, given that we will need to decode it back to buffer here. So final solution involves chancing this API to accept map of buffers, and chancing callers.
But for staging, you should feel free to simply decode here base64 to get a buffer and open an issue to follow up to remove needless conversions back and forth